docs/lib/predictions/fragments/ios/getting-started.md

```diff
@@ -137,13 +137,19 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 		}
 
 		print("AWSMobileClient initialized, userstate: \(userState)")
```

Comment:
Shouldn’t we add the `AWSMobileClient.initialize` call with the print inside the closure?

Reply:
Sorry... hmm not following what you are suggesting?

Reply:
ignore, i am dumb

Reply:
No you're not! :)